modules/developer_manual/pages/_partials/app/fundamentals/complete-info.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
+<info>
+    <!-- Mandatory fields -->
+    <id>yourappname</id>
+    <name>Your App</name>
+    <description>Your application description</description>
+    <version>1.0</version>
+    <licence>AGPL</licence>
+    <screenshot small-thumbnail="https://raw.githubusercontent.com/foo/myapp/master/screenshots/thumb.png"
+        >https://raw.githubusercontent.com/foo/myapp/master/screenshots/big.png</screenshot>
+    <!-- Category values available at: https://marketplace.owncloud.com/ajax/categories -->
+    <category>A category for the application. </category>
+    <summary>A summary of the application's purpose (max 90 chars)</summary>
+
+    <types>
+        <filesystem/>
+    </types>
+
+    <documentation>
+        <user>https://doc.owncloud.com/server/latest/user_manual/pim/contacts.html</user>
```

Comment:
do there urls exist?

Reply:
Does the URL exist? Yes.

Reply:
No idea if we should reference the docs of he contacts app for this example app being named yourapp .... no idea - up to you @settermjd